app/models/tag.rb

```diff
@@ -1,6 +1,6 @@
 class Tag < ActiveRecord::Base
   belongs_to :user
   has_and_belongs_to_many :posts
-
   validates_presence_of :user, :title
+  validates :title, uniqueness: true
```

Comment:
I think it makes sense to not create duplicated tags, but we may check this with Andrew to make sure this is the expected behaviour